### ISSUE TITLE
Add all pages to the pa11y scan

### ISSUE THREAD

**Issue by pcraig3** [cds-snc/cra-alpha#45]
The pull request #41, brought in an accessibility scanner that we run on new PRs, but it only scans the first three URLs of the app (`"/"`, `"/login"`, `"/introduction"`).

We should be scanning all the pages once we have a sense that they are not going away anytime soon.

There's a small PR: #44 that shows how to add one new page. Future pages can be added in a similar way.

**Comment by pcraig3** [comment]
no
